experimental/table/client/pom.xml

```diff
@@ -67,6 +67,11 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-metastore</artifactId>
+      <version>3.0.0</version>
```

Comment:
maybe we should make sure that we use a consistent `hive-metastore` dependency in our project? see  server pom. I don't have a preference on which to use, as long as we are consistent

Reply:
@yuzhu  Since I use version 3.0.0 in the connector side, should we keep 3.0.0 here?

Reply:
yeah we should put the dependency into the parent, and use `dependencyManagement` declare it there. and only include it when necessary, in server and client.